trueos-handbook/introducing.rst

```diff
@@ -475,83 +478,129 @@ manager utilities.
    :class: longtable
 
    +------------+-----------+--------------+--------------------------------------------------------+
-   | Filesystem | Native to | Non-native   | Usage notes                                            |
-   |            |           | support type |                                                        |
+   | Filesystem | Native OS | Non-native OS| Usage notes                                            |
+   |            |           | support      |                                                        |
    +============+===========+==============+========================================================+
-   | Btrfs      | Linux     | none         |                                                        |
+   | Btrfs      | Linux     | none         | A modern copy on write (CoW) filesystem for the Linux  |
+   |            |           |              | OS. Btrfs is similar in nature to ZFS, and shares many |
+   |            |           |              | of the same ideas with how a file system should work.  |
+   |            |           |              | The maximum supported volume and file size is 16EB.    |
    +------------+-----------+--------------+--------------------------------------------------------+
-   | exFAT      | Windows   | none         | requires a license from Microsoft                      |
+   | EXT2       | Linux     | r/w support  | The successor to EXT. EXT2 was designed following the  |
+   |            |           | loaded by    | principles put forth in BSD's FFS. The first           |
+   |            |           | default      | commercial grade filesystem in Linux. The maximum      |
+   |            |           |              | supported volume size is 2TB to 32TB and the maximum   |
```

Comment:
I looked all of the filesystems up to find the max supported volume and file size. Some have official sites/docs, some I had to rely on Wikipedia. As Btrfs is in active development, the max volume/filesize can could/will change in the future. For a filesystem like fat16, probably not so much. Would it make sense to leave the max volume/file size for "old" filesystems, and link to the docs for a filesystem that is still actively developed? My thinking here was this may help someone that's trying to say copy a 6GB file to a 32GB flash drive that is formatted fat32 for whatever reason. I think the file would "successfully" copy, but then either be truncated or corrupted when trying to retrieve it.  I'll leave the entries as is until there is more discussion.